ext/node/polyfills/01_require.js

```diff
@@ -954,7 +974,7 @@ Module._extensions[".js"] = function (module, filename) {
   const content = ops.op_require_read_file(filename);
 
   if (StringPrototypeEndsWith(filename, ".js")) {
-    const pkg = ops.op_require_read_closest_package_json(filename);
+    const pkg = ops.op_require_read_package_scope(filename);
```

Comment:
This was the real issue why it threw an error. It took me a while to find it. The difference is that the former throws an error and can lead to a panic. The latter returns an `Option` which our code kinda expects here. I think this was just a typo as the function in node refers to "package scope" as well: https://github.com/nodejs/node/blob/38e6ac7b446bcf2be257fde6763d1da7d2104295/lib/internal/modules/cjs/loader.js#L1307